README/ReleaseNotes/v634/index.md

```diff
@@ -57,6 +57,7 @@ The following people have contributed to this new version:
  Garima Singh, ETH,\
  Juraj Smiesko, CERN/RCS-PRJ-FC,
  Pavlo Svirin, National Technical University of Ukraine,\
+ Robin Syring, Leibniz University Hannover/ CERN/EP-SFT,\
```

Comment:
```suggestion
 Robin Syring, Leibniz University Hannover, CERN/EP-SFT,\
```
I think this is the style that we follow for University + CERN.

By the way, some people forgot the trailing backspace at the end of their entry in this list. It's needed for all but the last entry. Can you maybe fix this if you already edit this file?